dispersy.py

```diff
@@ -1246,18 +1291,18 @@ def check_double_member_and_global_time(unique, times, message):
 
                                 if have_packet < message.packet:
                                     # replace our current message with the other one
-                                    self._database.execute(u"UPDATE sync SET member = ?, packet = ? WHERE id = ?",
+                                    yield self._database.stormdb.execute(u"UPDATE sync SET member = ?, packet = ? WHERE id = ?",
                                                            (message.authentication.member.database_id, buffer(message.packet), packet_id))
 
-                                    return DropMessage(message, "replaced existing packet with other packet with the same payload")
+                                    returnValue(DropMessage(message, "replaced existing packet with other packet with the same payload"))
 
-                                return DropMessage(message, "not replacing existing packet with other packet with the same payload")
+                                    returnValue(DropMessage(message, "not replacing existing packet with other packet with the same payload"))
```

Comment:
I think you accidentally inserted a tab here (these two `returnValue` statements should not be on the same level).